components/header.css

```diff
@@ -1,18 +1,19 @@
 header {
   display: flex;
   justify-content: center;
-  flex-wrap: wrap;
+  /* flex-wrap: wrap; */
   border-radius: 10px;
   border: solid 2px var(--color-scarlet);
   background-color: var(--color-cream);
   position: fixed;
-  width: 97%;
-  text-align: center;
+  width: 95%;
+  /* text-align: center; */
   z-index: 1;
   top: 0;
-  margin-top: 20px;
+  margin-top: 0.2rem;
```

Comment:
Before, you had 20px and in the rest of your code, you always nearly calculated px to rem. 20px are not 0.2rem, maybe this is the problem with the header?

If you have an issue with the positioning, did you check the header with the dev tools? maybe you can find the problem there :)

I would be very interested in the issue, if you want to, we could look at it together :)

Reply:
Thanks for that very kind offer! I would love to look at it together. Perhaps tomorrow afternoon (Wed. 09 Aug) during autonomous learning?

Reply:
Perfect!